### ISSUE TITLE
Inference Submission: Top Level Dirs

### ISSUE THREAD

**Issue by bitfort** [mlperf/policies#9]
The rules appear want to the submitter's name as the top level directory. The checking script wants "open", "closed", ... as the top level.

**Comment by guschmue** [comment]
Yes, the submitter's repo is setup this way and I wanted to make sure we check the PR in a form that it can be merged into the repo. 
I think we should change the submission guide to match the repo.
Fyi: @petermattson

**Comment by psyhtest** [comment]
I like @guschmue's suggestion.

**Comment by guschmue** [comment]
Maybe let me be specific - the submission guide wants
```submiter-org/```
at the top. The checker wants
```division/submiter-org/```

There are 2 reasons why it needs to be ```division/submiter-org/```:
1. a submitter might want to submit for open and closed
2. the repo uses ```division/submiter-org/``` so if we receive PRs with ```submiter-org/``` one would need to merge it and than manually move it and send a second PR. Considering we are expecting a log of submissions ... lots of labor.
